reference/marketing.v2.yml

```
@@ -1511,11 +1511,12 @@ components:
           purchase_date:
             type: string
             description: Date the gift certificate was purchased. If not assigned,
```

```suggestion
            description: Date the gift certificate was purchased. If not assigned, this will be set to today’s date. Date displays in the [RFC 2822](https://www.rfc-editor.org/rfc/rfc2822#section-3.3) timestamp format.
```
put description on a single line, add link to spec.